packages/app-headless-cms/src/admin/components/Droppable.tsx

```diff
@@ -33,10 +33,15 @@ export interface DroppableProps {
     onDrop?: OnDropCallable;
 }
 
+type DragObject = {
+    item: any;
```

Comment:
Really no type for item that can be used?

Reply:
@brunozoric `item` described as `any` in all interfaces inside of Droppable.tsx

I will try to find a better type.
Maybe it should be described as `DragSource`.